tractor/_trionics.py

```diff
@@ -274,13 +287,13 @@ def do_hard_kill(proc):
 
 
 @asynccontextmanager
-async def open_nursery(supervisor=None):
+async def open_nursery() -> typing.AsyncGenerator[None, ActorNursery]:
```

Comment:
Got the arguments backwards? Isn't the second argument the send type? I see you yielding a nursery.